e2e/cypress/support/api/system.js

```diff
@@ -152,3 +152,16 @@ function uploadLicenseIfNotExist() {
         });
     });
 }
+
+Cypress.Commands.add('apiDeleteBrandImage', () => {
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/brand/image',
+        method: 'DELETE',
+        failOnStatusCode: false,
+    }).then((response) => {
+        // both deleted and not existing responses are valid
+        expect(response.status).to.be.oneOf([200, 404]);
+        return cy.wrap(response);
+    });
+});
```

Comment:
I missed to mention on my previous review to move this into a separate file `api/brand.js` and also the type at `api/brand.d.ts`. It is to match the folder structure in https://api.mattermost.com.

Reply:
Done! PTAL :)